src/clients/aggregator_client/api_types.rs

```diff
@@ -101,12 +101,33 @@ impl From<Option<i64>> for QueryType {
     }
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
+pub struct AuthenticationToken {
```

Comment:
Note that we use a Serde struct for this object here, and an internally-tagged enum in Janus. In follow-up work, I think it might make sense to treat the authentication token as an opaque JSON blob.

Reply:
Oh! In thinking this through I realized we never need to _use_ these tokens in this repo, only pass them between servers? In that case I can close #248 and I agree with this comment. Unless divviup-api needs to be the owner of acceptable tokens, we could just pass around a `serde_json::Value`

Reply:
That would work for the aggregator auth tokens, which are opaque to `divviup-api`, but not necessarily for collector auth tokens. The tension here is that the Janus aggregator API can express multiple token types, hence the `{ "type": <>, "token": <> }` structure, but `divviup-api` represents collector auth tokens as a string which implicitly should be used as Bearer tokens. So when providing collector auth tokens to subscribers, `divviup-api` needs to extract the `token` field of the `AuthenticationToken` structure.

We could make this less awkward but using the `AuthenticationToken` representation in the API `divviup-api` offers to its clients.